documentation/decisions/adr/00NN-frontend-testing.md

```diff
@@ -0,0 +1,98 @@
+# Front-end Testing & Coverage
+
+- **Status:** Proposed <!-- REQUIRED -->
```

Comment:
```suggestion
- **Status:** Accepted <!-- REQUIRED -->
```
The template no longer includes "Proposed" since an ADR PR isn't merged unless the decision is approved.

Reply:
Alright, thank you. In the future I'll just put accepted first then and not go through the process of updating it.